GitTfs/Core/GitTfsRemote.cs

```diff
@@ -695,10 +703,17 @@ private void quickFetch(ITfsChangeset changeset)
         private IEnumerable<ITfsChangeset> FetchChangesets(bool byLots, int lastVersion = -1)
         {
             int lowerBoundChangesetId;
+
+            // If we're starting at the Root side of a branch commit (e.g. C1) but there are invalid commits between C1 and the actual branch-side of the commit operation
```

Comment:
I'm sure it would... but that also means roughly 2/3 of a 1080p display is wasted when editing source through Visual Studio.
It feels like trying to fix a problem by treating a symptom (line width) rather than the problem (poor code review layout in GitHub).